notanorm/base.py

```diff
@@ -247,7 +265,8 @@ def query(self, sql, *args, factory=None):
             if factory:
                 row = factory(**row)
             else:
-                row = DbRow(row)
+                if type(row) is not DbRow:
```

Comment:
when would this be a `DbRow`?

Reply:
if the driver decides it knows how to build one up more efficiently than we do